proof/crefine/Move_C.thy

```diff
@@ -1493,4 +1493,15 @@ lemma from_to_bool_le_1_idem:
   apply (simp add: word_le_1_and_idem)
   done
 
+(* FIXME AARCH64: this is very specific and rather ugly, is it possible to generalise? *)
```

Comment:
kind of odd to move this into generic crefine, no?

Reply:
You're right, this should be in ArchMove_C, not Move_C